### PR TITLE
Changing Location from drop down to a string

### PR DESCRIPTION
**Pull request by japhari**
BE REQUIRED
OP-1396

Changes:
Change the way of displaying information about policy holders

### CODE REVIEW

File: src/components/PolicyHolderSearcher.js [hunk 1]
```diff
@@ -18,224 +18,309 @@ import { IconButton, Tooltip } from "@material-ui/core";
...
+        !!policyHolder.code && policyHolder.tradeName
+          ? `${policyHolder.code} ${policyHolder.tradeName}`
+          : "",
+      (policyHolder) =>
+        !!policyHolder.locations
+          ? `${policyHolder.locations.parent.parent.parent.name} 
+            ${policyHolder.locations.parent.parent.name} 
+            ${policyHolder.locations.parent.name}
+            ${policyHolder.locations.name}`
+          : "",
```
Comment (olewandowski1) [inline src/components/PolicyHolderSearcher.js:106]: @japhari Initial readOnly picker was displaying name of single location as well as it's code. Currently, you're displaying just a name of locations. It should be changed.

File: src/components/PolicyHolderUserSearcher.js [hunk 2]
```diff
@@ -36,310 +36,341 @@ import PolicyHolderPicker from "../pickers/PolicyHolderPicker";
...
+    const {
+      intl,
+      modulesManager,
+      rights,
+      onSave,
+      predefinedPolicyHolderId = null,
+    } = this.props;
+    const result = [
+      (policyHolderUser) =>
+        !!policyHolderUser.user ? `${policyHolderUser.user.username}` : "",
```
Comment (olewandowski1) [inline src/components/PolicyHolderUserSearcher.js:142]: @japhari Initial readOnly picker was displaying _user name_ as well as a _last name_. Currently, you're displaying just a username. The missing _last name_ parameter should be added.

**Review by olewandowski1 — CHANGES_REQUESTED** [review]
I left you some comments @japhari . Please refer to them.

File: src/components/PolicyHolderUserSearcher.js (hunk 2 above)
Comment (japhari) [reply src/components/PolicyHolderUserSearcher.js:142]: Okay working on it

**Review by olewandowski1 — APPROVED** [review]
LGTM @japhari @dborowiecki

**Pull request merged**